File: Tests/AnnotationsWithMultipleOutputs/SecondType.cs
using KY.Generator;

namespace AnnotationsWithMultipleOutputs
{
    [GenerateAngularModel("Output\\Second")]
    [GenerateNoHeader]
    public class SecondType
    {
        public string StringProperty { get; set; }
        public SubType SubTypeProperty { get; set; }
    }
}

File: Tests/AnnotationsWithMultipleOutputs/Output/Second/second-type.ts
/* eslint-disable */
// tslint:disable

import { SubType } from "./sub-type";

export class SecondType {
    public stringProperty: string;
    public subTypeProperty: SubType;

    public constructor(init: Partial<SecondType> = undefined) {
        Object.assign(this, init);
    }
}

// outputid:352b1947-a770-4737-be18-608a78c130ad
